spec/requests/api/v1/follow_requests_spec.rb

```diff
@@ -21,7 +21,8 @@
         a_hash_including(
           id: account.id.to_s,
           username: account.username,
-          acct: account.acct
+          acct: account.acct,
+          follow_request_created_at: FollowRequest.find_by(account_id: account.id).created_at
```

Comment:
Similar feedback here as in controller - make sure it's the one actually relevant to account requesting the endpoint.